src/scripts/ui/FillPlayerTradeModal.tsx

```diff
@@ -82,47 +88,196 @@ export function FillPlayerTradeModal({ tradeId, xy }: { tradeId: string; xy?: Ti
       return true;
    };
 
-   const calculateMaxFill = () => {
+   // major changes 2025-02-xx
+   // param "hint" can be "greedy" or "fast" to tweak some limits
+   // param "maxError" reserved for future use, to set tolerance for fluctuating, erronous or invalid values
+   const calculateMaxFill = (hints = "auto", maxError = 0.03) => {
```

Comment:
Again, `hints` should not be included. Also the magic number `0.03` should not be included